src/app/components/layout/header/header.component.html

```diff
@@ -1,75 +1,61 @@
+        ></path>
+        <image
+          src="/assets/images/moj-logotype-crest.png"
+          xlink:href=""
+          class="moj-header__logotype-crest-fallback-image"
+          width="40"
+          height="40"
+        ></image>
+      </svg>
+      <a class="moj-header__link moj-header__link--organisation-name" href="#">HMCTS</a>
+      <a class="moj-header__link moj-header__link--service-name" href="#">DARTS</a>
     </div>
-    <div class="govuk-header__content">
-      <a href="#" class="govuk-header__link govuk-header__service-name"> DARTS portal </a>
-      <nav aria-label="Menu" class="govuk-header__navigation">
-        <button
-          type="button"
-          class="govuk-header__menu-button govuk-js-header-toggle"
-          aria-controls="navigation"
-          aria-label="Show or hide menu"
-          hidden
-        >
-          Menu
-        </button>
-        <ul id="navigation" class="govuk-header__navigation-list">
-          <li class="govuk-header__navigation-item">
-            <a class="govuk-header__link" routerLink="/inbox" routerLinkActive="active" ariaCurrentWhenActive="page"
-              >Inbox</a
-            >
+    <div class="moj-header__content">
+      <nav class="moj-header__navigation" aria-label="Account navigation">
+        <ul class="moj-header__navigation-list">
+          <li class="moj-header__navigation-item">
+            <a class="moj-header__navigation-link" href="#">Sign out</a>
           </li>
-          <li class="govuk-header__navigation-item">
-            <a class="govuk-header__link" routerLink="/audios" routerLinkActive="active" ariaCurrentWhenActive="page"
-              >My Audios</a
-            >
+        </ul>
+      </nav>
+    </div>
+  </div>
+</header>
+<div class="moj-primary-navigation">
+  <div class="moj-primary-navigation__container">
+    <div class="moj-primary-navigation__nav">
+      <nav class="moj-primary-navigation" aria-label="Primary navigation">
+        <ul class="moj-primary-navigation__list">
+          <li class="moj-primary-navigation__item">
+            <a class="moj-primary-navigation__link" aria-current="page" href="inbox">Inbox</a>
+          </li>
+          <li class="moj-primary-navigation__item">
+            <a class="moj-primary-navigation__link" href="audios">My Audios</a>
```

Comment:
Can you please make the nav items with with angular routes like the previous ones did, something like 

```suggestion
            <a class="moj-primary-navigation__link" routerLink="/audios" routerLinkActive="active" ariaCurrentWhenActive="page">My Audios</a>
```